privaterelay/templates/base.html

```diff
@@ -24,9 +27,11 @@
     {% endblock %}
 
     {% include "includes/footer.html" %}
-    <script type="text/javascript" src="{% static 'js/clipboard.min.js' %}" charset="utf-8"></script>
-    <script src="{% static 'js/fx-bento.js' %}"></script>
-    <script src="{% static 'js/app.js' %}"></script>
-    <script src="{% static 'js/analytics.js' %}"></script>
+    {% compress js %}
+      <script type="text/javascript" src="{% static 'js/clipboard.min.js' %}" charset="utf-8"></script>
```

Comment:
Not sure if this is a legit concern, but...
Should we _re_-minify the clipboard.min.js file?
Not sure if this should be removed and we just load two .js files.